.github/workflows/ci-cd.yml

```
@@ -16,9 +16,43 @@ jobs:
+    env:
+      ENVIRONMENT: testing
+      DATABASE_HOST: localhost
+      DATABASE_PORT: 5432
+      DATABASE_USER: postgres
+      DATABASE_PASSWORD: postgres
+      DATABASE_DBNAME: celebrum_test
+      REDIS_HOST: localhost
+      REDIS_PORT: 6379
+      CCXT_SERVICE_URL: http://localhost:3001
+      CONFIG_PATH: configs/config.ci.yml
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)